android/src/main/java/com/rnlib/adyen/AdyenPaymentModule.kt

```diff
@@ -523,7 +525,8 @@ class AdyenPaymentModule(private var reactContext : ReactApplicationContext) : R
     override fun onNewIntent(intent: Intent?) {
         Log.d(TAG, "onNewIntent")
         if (intent?.hasExtra(AdyenComponent.RESULT_KEY) == true) {
-            Log.d(TAG,intent.getStringExtra(AdyenComponent.RESULT_KEY))
+            // TODO: Define what to do if [intent.getStringExtra(AdyenComponent.RESULT_KEY)] is null
```

Comment:
Does `if (intent?.hasExtra(AdyenComponent.RESULT_KEY) == true) {` not check it is not null?